### PR TITLE
[RW-7258][risk=low] Fix PD e2e Test

### PR DESCRIPTION
**Pull request by YintongMa**
In this PR, 
1. Fix the e2e issue from PD UI Changes
   - Remove pick Disk operation in dataproc -> gce
   - Select specific deletion option by deletion radio button
2. Fix one small bug of dataproc deletion panel

### CODE REVIEW

File: ui/src/app/pages/analysis/runtime-panel.tsx [hunk 1]
```diff
@@ -61,6 +61,7 @@ import {Dropdown} from 'primereact/dropdown';
 import {InputNumber} from 'primereact/inputnumber';
 import * as React from 'react';
 import {validate} from 'validate.js';
+import {RadioButton} from "../../components/inputs";
```
Comment (yonghaoy) [inline ui/src/app/pages/analysis/runtime-panel.tsx:64]: relative import is not encouraged, we should use absolute path

**Review by yonghaoy — CHANGES_REQUESTED** [review]
Please update PR description, that is not just fixing e2e tests, it also changing some product behaviour.

File: ui/src/app/pages/analysis/runtime-panel.tsx [hunk 2]
```diff
@@ -245,8 +246,13 @@ export const ConfirmDeleteUnattachedPD = ({onConfirm, onCancel}) => {
 
     <div style={styles.confirmWarning}>
       <h3 style={{...styles.baseHeader, ...styles.bold, gridColumn: 1, gridRow: 1}}>
-        <label ><input type= 'radio' checked={deleting === true} onChange={() => setDeleting(true)}/>
-          Delete persistent disk</label></h3>
+        <div key='Delete persistent disk' style={{display: 'inline-block', marginRight: '0.5rem'}}>
```
Comment (calbach) [inline ui/src/app/pages/analysis/runtime-panel.tsx:249]: key is mostly used for distinguishing elements of a list

To get an element via Puppeteer, I would either use the label text, or introduce a `data-test-id` here. @aweng98 may be able to advise on which approach is preferred.

**Review by yonghaoy — APPROVED** [review]

File: ui/src/app/pages/analysis/runtime-panel.tsx (hunk 2 above)
Comment (aweng98) [reply ui/src/app/pages/analysis/runtime-panel.tsx:249]: attribute `data-test-id` is preferred for e2e automation.

File: e2e/app/component/runtime-panel.ts [hunk 3]
```diff
@@ -6,6 +6,7 @@ import Button from 'app/element/button';
 import NotebookPreviewPage from 'app/page/notebook-preview-page';
 import BaseHelpSidebar from './base-help-sidebar';
 import { logger } from 'libs/logger';
+import RadioButton from "../element/radiobutton";
```
Comment (aweng98) [inline e2e/app/component/runtime-panel.ts:9]: Change to `app/element/radiobutton`. It will resolve lint error.

**Review by aweng98 — COMMENTED** [review]
Have you ran `test-nightly` locally to verify all tests passed?

File: ui/src/app/pages/analysis/runtime-panel.tsx [hunk 4]
```diff
@@ -245,8 +246,13 @@ export const ConfirmDeleteUnattachedPD = ({onConfirm, onCancel}) => {
 
     <div style={styles.confirmWarning}>
       <h3 style={{...styles.baseHeader, ...styles.bold, gridColumn: 1, gridRow: 1}}>
-        <label ><input type= 'radio' checked={deleting === true} onChange={() => setDeleting(true)}/>
-          Delete persistent disk</label></h3>
+        <div data-test-id='Delete unattached pd' style={{display: 'inline-block', marginRight: '0.5rem'}}>
```
Comment (calbach) [inline ui/src/app/pages/analysis/runtime-panel.tsx:249]: nit: following the pattern for other data-test-ids, I would just use hyphenated, no spaces: e.g. `delete-unattached-pd`

**Review by calbach — APPROVED** [review]

**Comment by YintongMa** [comment]
> Have you ran `test-nightly` locally to verify all tests passed?

I didn't run test-nightly on all tests locally. I verified on the dataproc-to-gce and gce-to-dataproc nightly tests, which have PD related issues.

**Review by aweng98 — APPROVED** [review]
Thank you.

**Pull request merged**